### PR TITLE
change langchain version to 0.1.0

### PR DESCRIPTION
**Pull request by vincent0318**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
 logger.addHandler(ch)
 
 class CourseLangChain():
-  def __init__(self, pickleFile="vectorstore.pkl", modelFile="model/chinese-alpaca-2-13b.Q8_0.gguf", cli=False) -> None:
```
Comment (andyjjrt) [inline main.py:29]: Dont change class preset parameter, if you want to test other model, you should pass through constructor

File: requirement.txt [hunk 2]
```diff
@@ -11,4 +11,5 @@ bitsandbytes==0.41.0
 tokenizers==0.13.3
 accelerate==0.21.0
 llama_cpp_python==0.2.11
-openai-whisper==20231106
\ No newline at end of file
+openai-whisper==20231106
+sqlalchemy==2.0.29
```
Comment (andyjjrt) [inline requirement.txt:15]: what's this?

**Review by andyjjrt — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (vincent0318) [reply main.py:29]: sorry~forgot to change back to the old one(alpaca-2-13b)

File: requirement.txt (hunk 2 above)
Comment (vincent0318) [reply requirement.txt:15]: SQLAlchemy is a popular SQL toolkit and Object Relational Mapping (ORM) system for Python.
After you update langchain, SQLAlchemy will be a requirement library.

As for openai-whisper, if I want to run app.py in my computer this module is necessary.

**Pull request merged**